presto-main/src/main/java/com/facebook/presto/metadata/QualifiedObjectName.java

```diff
@@ -85,6 +86,11 @@ public QualifiedTablePrefix asQualifiedTablePrefix()
         return new QualifiedTablePrefix(catalogName, schemaName, objectName);
     }
 
+    public CatalogSchemaName asCatalogSchemaName()
+    {
+        return new CatalogSchemaName(catalogName, schemaName);
```

Comment:
This doesn't return "another way to refer to `this`", so method should not be named `as*`. Rename to `getCatalogSchemaName`

Reply:
I fixed that. Thanks